### PR TITLE
Added Subscription section; fixed inaccurate revenue share numbers

### PR DESCRIPTION
**Pull request by igorin**
## Purpose of this pull request

This pull request (PR):
- Adds Subscription content under Marketplace Sellers section
- updates the revenue share percentages on relevant Marketplace Dev Docs pages, and includes mentions of subscription sales transactions under Manage Sales.

## Affected DevDocs pages

- [Revenue Share](https://devdocs.magento.com/marketplace/sellers/revenue-share.html)
- [Manage Sales](https://devdocs.magento.com/marketplace/sellers/sales.html)
- [Subscriptions](https://devdocs.magento.com/marketplace/sellers/subscriptions/subscriptions.html)
- [Buying Subscription-based Extensions](https://devdocs.magento.com/marketplace/sellers/subscriptions/buying-subscriptions.html)
- [Selling Subscription-based Extensions](https://devdocs.magento.com/marketplace/sellers/subscriptions/selling-subscriptions.html)

whatsnew
Added new topics about [buying](https://devdocs.magento.com/marketplace/sellers/subscriptions/buying-subscriptions.html) and [selling](https://devdocs.magento.com/marketplace/sellers/subscriptions/selling-subscriptions.html) subscription-based extensions.

### CODE REVIEW

**Review by BarnyShergold — APPROVED** [review]
Just one suggestion but in case you feel it needs to stand then I'll approve.

File: src/marketplace/sellers/revenue-share.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ title: Revenue Share
...
 
-All Commerce Marketplace sales are subject to a 70/30 revenue share split, with 70% of the revenue for the provider and 30% for Magento. Sales subject to the revenue share include both paid extensions and converted indirect leads from software as a service (SaaS), or subscription-based products such as a free extension bundled with a paid subscription. All product/service bundles are subject to the same 70/30 revenue share business model.
+All Commerce Marketplace sales are subject to a 85/15 revenue share split, with 85% of the revenue for the provider and 15% for Adobe. Sales subject to the revenue share include both paid extensions and converted indirect leads from software as a service (SaaS), or subscription-based products such as a subscription-based paid extension or a free extension bundled with a paid subscription. All product/service bundles are subject to the same 85/15 revenue share business model.
```
[the remark this answers is not in the document]
Comment (igorin) [reply src/marketplace/sellers/revenue-share.md:8]: let's keep it as is for now and enjoy the extra SEO for "Marketplace Subscriptions"

File: src/marketplace/sellers/subscriptions/selling-subscriptions.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+group: marketplace-sellers
+title: Selling Subscription-based Extensions
+redirect_from: /marketplace/sellers/subscriptions/selling-subscriptions.html
+---
+Starting December 2021, Adobe Commerce Marketplace provides extension vendors an opportunity to sell their extensions under an auto-renewing subscription model.
+
+## Setting up subscription-based billing
+While submitting a new extension to the marketplace, you need to select the Pricing Model for the extension. The Pricing model can only be selected when the initial version is being set up. Once the initial version is created on the Developer Portal, the pricing model cannot be changed for this SKU. Choose Subscription to set the SKU under subscription license, and set the subscription price.
```
Comment (jeff-matthews) [inline src/marketplace/sellers/subscriptions/selling-subscriptions.md:9]:
```suggestion

When submitting a new extension to the marketplace, you must select the pricing model for the extension. The pricing model can only be selected when the initial version is being set up. After the initial version is created on the Developer Portal, the pricing model cannot be changed for this SKU. Choose Subscription to set the SKU under subscription license, and set the subscription price.
```

File: src/marketplace/sellers/subscriptions/selling-subscriptions.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+[]({{ site.baseurl }}/marketplace/sellers/images/subscriptions-pricing-model.png){: .zoom}
+
+{: .bs-callout .bs-callout-info}
+**Note:** at this time, only Annual subscription licenses are supported.
+
+There are no subscription-specific settings other than Pricing Model.
+
+## Changing existing extension from flat fee to subscription
+You may elect to change licensing for their existing extension from Flat Fee to Subscription. In order to do so, you will need to remove their original extension from the Marketplace and replace it with a new SKU sold under a subscription license.
```
Comment (jeff-matthews) [inline src/marketplace/sellers/subscriptions/selling-subscriptions.md:19]:
```suggestion

You may elect to change licensing for an existing extension from Flat Fee to Subscription. In order to do so, you must remove the original extension from the Marketplace and replace it with a new SKU sold under a subscription license.
```

File: src/marketplace/sellers/subscriptions/selling-subscriptions.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+We recommend starting by submitting a new SKU. This allows new sales operations and marketing activities to start generating revenue right away. Follow the section above to set up a new SKU under a subscription license.
+You can then either remove the original extension unconditionally, or mark the new extension as a replacement. This will trigger a 180-day grace period for merchants who have purchased the original extension in the past.
```
Comment (jeff-matthews) [inline src/marketplace/sellers/subscriptions/selling-subscriptions.md:21]:
```suggestion
You can then either remove the original extension unconditionally, or mark the new extension as a replacement. This triggers a 180-day grace period for merchants who have purchased the original extension.
```

File: src/marketplace/sellers/subscriptions/selling-subscriptions.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+To make that choice, use the "Remove from Store" button on your original extension.
```
Comment (jeff-matthews) [inline src/marketplace/sellers/subscriptions/selling-subscriptions.md:22]:
```suggestion
To remove an extension from the Marketplace, use the **Remove from Store** button on your original extension.
```

File: src/marketplace/sellers/subscriptions/selling-subscriptions.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+[]({{ site.baseurl }}/marketplace/sellers/images/subscriptions-remove-from-store.png){: .zoom}
+
+If you decide to use the first option (unconditional removal), your original extension will disappear from the Marketplace and no other action will be taken.
+If you select the second option (replace), you will be able to select a replacement extension from the list of your active SKUs. The 180-day grace period will be applied to the replacement extension for customers who have purchased the original extension.
+
+[]({{ site.baseurl }}/marketplace/sellers/images/subscriptions-confirm-remove.png){: .zoom}
+[]({{ site.baseurl }}/marketplace/sellers/images/subscriptions-remove-replace.png){: .zoom}
+
+{: .bs-callout .bs-callout-info}
+**Important:** once the extension is removed from the Marketplace, the customers will still be able to download the latest version of the module, but you will no longer be able to submit new versions. Plan your rollout accordingly.
```
Comment (jeff-matthews) [inline src/marketplace/sellers/subscriptions/selling-subscriptions.md:33]:
```suggestion
{:.bs-callout-warning}
After the extension is removed from the Marketplace, customers will still be able to download the latest version of the module, but you will no longer be able to submit new versions. Plan your rollout accordingly.
```

File: src/marketplace/sellers/subscriptions/selling-subscriptions.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+**Note:** the grace period will only apply if the replacement extension is a subscription extension.
```
Comment (jeff-matthews) [inline src/marketplace/sellers/subscriptions/selling-subscriptions.md:35]:
```suggestion
{:.bs-callout-info}
The grace period only applies if the replacement extension is a subscription extension.
```

File: src/marketplace/sellers/subscriptions/selling-subscriptions.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+## Pricing management
+You can change your extension price every time you submit a new version of your extension.
```
Comment (jeff-matthews) [inline src/marketplace/sellers/subscriptions/selling-subscriptions.md:38]:
```suggestion

You can change your extension price every time you submit a new version of your extension.
```

File: src/marketplace/sellers/subscriptions/selling-subscriptions.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+Pricing model cannot be changed once at least one version of the extension is initially created on the DevPortal.
```
Comment (jeff-matthews) [inline src/marketplace/sellers/subscriptions/selling-subscriptions.md:39]:
```suggestion
Pricing model cannot be changed after at least one version of the extension is initially created on the Developer Portal.
```

File: src/marketplace/sellers/subscriptions/selling-subscriptions.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+The existing customers will be notified that the price of the extension has been changed.
```
Comment (jeff-matthews) [inline src/marketplace/sellers/subscriptions/selling-subscriptions.md:40]:
```suggestion
Existing customers will be notified that the price of the extension has changed.
```

File: src/marketplace/sellers/subscriptions/selling-subscriptions.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+## Tracking sales
+You can see the subscription sales as a regular transactions in the Developer Portal. We are working on improving developer sales tracking functionality and will update documentation when it will be ready.
```
Comment (jeff-matthews) [inline src/marketplace/sellers/subscriptions/selling-subscriptions.md:43]:
```suggestion

You can see subscription sales as regular transactions in the Developer Portal. We are working on improving developer sales tracking functionality and will update documentation when it's ready.
```

File: src/marketplace/sellers/subscriptions/selling-subscriptions.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+
+## Cancellations and refunds
+A customer who purchased an extension under a subscription license can request a refund within 25 days. The customer will need to open a Support ticket for this request.
```
Comment (jeff-matthews) [inline src/marketplace/sellers/subscriptions/selling-subscriptions.md:46]:
```suggestion

A customer who purchased an extension under a subscription license can request a refund within 25 days. The customer must open a Marketplace Support ticket for this request.
```

File: src/marketplace/sellers/subscriptions/selling-subscriptions.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+Customers can cancel their extension subscription at any time.
+
+**Important:** Customers can only re-subscribe to a cancelled extension subscription once. Repeated cancellations and re-subscriptions to an extension are not supported.
```
Comment (jeff-matthews) [inline src/marketplace/sellers/subscriptions/selling-subscriptions.md:49]:
```suggestion
{:.bs-callout-info}
Customers can only re-subscribe to a cancelled extension subscription once. Repeated cancellations and re-subscriptions to an extension are not supported.
```

**Review by jeff-matthews — CHANGES_REQUESTED** [review]

**Comment by igorin** [comment]
Committed all changes - thanks @jeff-matthews for the review!

File: src/marketplace/sellers/subscriptions/subscriptions.md [hunk 14]
```diff
@@ -0,0 +1,20 @@
+---
+group: marketplace-sellers
+title: Extension Subscriptions
+redirect_from: /marketplace/sellers/subscriptions/subscriptions.html
```
[the remark this answers is not in the document]
Comment (igorin) [reply src/marketplace/sellers/subscriptions/subscriptions.md:4]: how about I rename the file to extension-subscriptions.md and redirect to it from /subscriptions/extension-subscriptions.html?

**Comment by jeff-matthews** [comment]
@igorin, there are actually a bunch more. GitHub collapses them after they reach a certain number. I recommend going to the **Files changed** tab at the top of the PR page and adding all suggestions to "batch".

File: src/marketplace/sellers/subscriptions/subscriptions.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (jeff-matthews) [reply src/marketplace/sellers/subscriptions/subscriptions.md]: renaming it works for me. remove the redirect. it's not necessary for new, unpublished files.

File: src/marketplace/sellers/subscriptions/selling-subscriptions.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (igorin) [reply src/marketplace/sellers/subscriptions/selling-subscriptions.md]: Resolved

**Review by jeff-matthews — APPROVED** [review]
well done! thanks for the PR @igorin. i'll import into the delivery repo and merge.

**Pull request merged**